modules/tinymce/src/plugins/lists/test/ts/browser/StyleTest.ts

```diff
@@ -0,0 +1,67 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import * as InsertNewLine from 'tinymce/core/newline/InsertNewLine';
+import Plugin from 'tinymce/plugins/lists/Plugin';
+
+describe('browser.tinymce.plugins.lists.StyleTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    plugins: 'lists',
+    toolbar: false,
+    indent: false,
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Plugin ], true);
+
+  it('TINY-10316: Create a new `li` from a nexted list should preserve the style', () => {
```

Comment:
Is `nexted` a word? Also since this is altering the core not the list plugin I expect this test to be added in the core under the newline tests or as a separate test file for list new lines. Also it shouldn't have to enable the `lists` plugin in that test. I guess we could still have a test here for the list plugin just to make sure that the enter key works as expected in the list plugin as well.

Reply:
sorry it was `nested`, I refactored to have 2 different tests and one time use `lists` and the other without plugins